packages/devtools/lib/src/ui/analytics.dart

```diff
@@ -0,0 +1,282 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@JS()
+library gtags;
+
+import 'dart:async';
+import 'package:js/js.dart';
+
+import '../ui/gtags.dart';
+import '../ui/ui_utils.dart';
+
+// Dimensions1 AppType values:
+const String appTypeFlutter = 'flutter';
+const String appTypeWeb = 'web';
+// Dimensions2 BuildType values:
+const String buildTypeDebug = 'debug';
+const String buildTypeProfile = 'profile';
+// Dimensions3 PlatformType values:
+const String platformTypeAndroid = 'android_flutter';
+const String platformTypeIOS = 'ios_flutter';
+const String platformTypeFuchsia = 'fuchsia';
+const String platformTypeLinux = 'linux';
+const String platformTypeMac = 'mac';
+const String platformTypeWindows = 'windows';
+// Dimension4 devToolsPlatformType values:
+const String devToolsPlatformTypeMac = 'MacIntel';
+const String devToolsPlatformTypeLinux = 'Linux';
+const String devToolsPlatformTypeWindows = 'Windows';
+// Start with Android_n.n.n
+const String devToolsPlatformTypeAndroid = 'Android_';
+// Dimension5 devToolsChrome starts with
+const String devToolsChrome = 'Chrome/'; // starts with and ends with n.n.n
+const String devToolsChromeIos = 'Crios/'; // starts with and ends with n.n.n
```

Comment:
do you mean Cros not Crios?
Similarly, devtoolsChromeIos --> devtoolsChromeOs

nit: consider  classes with static members for these instead of getting reallyLongNamesWithLotsofPrefixes. That would also make it easier to see which entries are grouped together and give a place to add a comment describing the block.
E.g.
class Platform {
static final String mac = ...;
}
class DevToolsPlatform {
...
}

Reply:
Added `&gtags=enabled | disabled`.  Note to use ga-confirm= then `&gtags=enabled` must also be set.
The default value for `&gtags=` if not set is disabled.